### PR TITLE
[FLYW-43] Center flag

### PR DESCRIPTION
**Pull request by hkethi002**
Ref #1183 

### Review Checklist

- Tests were added to cover all code changes
- Documentation was added / updated
- Code and tests follow standards in CONTRIBUTING.md

### CODE REVIEW

File: api/auth/groupauth.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def f(method, _id=None, query=None, payload=None, projection=None):
                 pass
             elif method in ['DELETE', 'POST']:
                 handler.abort(403, 'not allowed to perform operation')
-            elif _get_access(handler.uid, group) >= INTEGER_PERMISSIONS['admin']:
+            elif _get_access(handler.uid, group) >= INTEGER_PERMISSIONS['admin'] and 'edition' not in payload:
```
Comment (kofalt) [inline api/auth/groupauth.py]: Help me out, why does edition need to be in the payload here?

Comment (hkethi002) [reply api/auth/groupauth.py]: Modifying a groups edition requires site_admin (user.root == True) so it should get caught in [line 14](https://github.com/flywheel-io/core/pull/1477/files/959530b564a7b96dfc54c15636ba5160940ebc2a#diff-6c242292143e0746e65d38b88fde219aR14)

File: api/dao/hierarchy.py [hunk 2]
```diff
@@ -39,6 +39,22 @@ def get_container(cont_name, _id):
         '_id': _id,
     })
 
+def is_edition(edition, cont_name, _id):
+    container = get_container(cont_name, _id)
```
Comment (ehlertjd) [inline api/dao/hierarchy.py]: Take a container parameter to prevent extra retrievals whenever we check edition

File: api/dao/hierarchy.py [hunk 3]
```diff
@@ -39,6 +39,22 @@ def get_container(cont_name, _id):
...
+    if cont_name != 'groups':
+        group = container.get('parents', {}).get('group')
+    else:
+        group = _id
+    if group:
+        editions = get_container('groups', group).get('edition', [])
```
Comment (ehlertjd) [inline api/dao/hierarchy.py]: We really should turn on `redis` or `memcached` for this kind of stuff

Comment (hkethi002) [reply api/dao/hierarchy.py]: ok

**Comment by nagem** [comment]
Requirements have changed, this is no longer in line with expectation for this feature.

**Pull request closed without merging**